### PR TITLE
fix: latest commits from beta

### PR DESCRIPTION
**Pull request by Parimala032**
**This Pull Request addresses** 
latest commits from the beta branch merge into next branch till date (20-03-2024)
minor changes in the plugin-devices and plugin-metrics
**By making the fallowing changes**
Cherry-picking the changes 
Marked browser test as broken in internal-plugin-device
Removed the browser test's command from internal-plugin-metrics as there are no browser tests for plugin-metrics

### CODE REVIEW

**Review by Shreyas281299 — APPROVED** [review]
LGTM

**Review by Shreyas281299 — CHANGES_REQUESTED** [review]
Reverting approval, lets add the latest commits from beta as well.

**Review by Shreyas281299 — APPROVED** [review]
LGTM

File: packages/@webex/internal-plugin-metrics/package.json [hunk 1]
```diff
@@ -52,7 +52,6 @@
     "build:src": "webex-legacy-tools build -dest \"./dist\" -src \"./src\" -js -ts -maps && yarn build",
     "deploy:npm": "yarn npm publish",
     "test": "yarn test:style && yarn test:unit && yarn test:integration && yarn test:browser",
-    "test:browser": "webex-legacy-tools test --integration --unit --runner karma",
```
Comment (sreenara) [inline packages/@webex/internal-plugin-metrics/package.json:55]: This is removed because there are no browser tests for the metrics plugin.

**Review by sreenara — APPROVED** [review]

**Pull request merged**